packages/vanilla/src/index.ts

```diff
@@ -76,7 +76,13 @@ export const loadParallel = (config: ParallelConfig): Promise<Parallel | null> =
     return new Promise((resolve) => {
       // Resolve the initialized Parallel object upon init
       const onInit = () => resolve(Parallel)
-      Parallel.init({ ...config, on_init: onInit })
+      // The `raw_config` option lets you pass in extra values that aren't described in the ParallelConfig type
+      // such as upcoming features that are being tested.
+      const baseConfig = config.raw_config ?? {}
+      const rawlessConfig = { ...config }
+      delete rawlessConfig.raw_config
+      const parallelConfig = { ...baseConfig, ...rawlessConfig }
+      Parallel.init({ ...parallelConfig, on_init: onInit })
```

Comment:
There are a few additional params that aren't currently in the ParallelConfig - 
 * expected_entity_id
 * expected_entity_type
 * expected_business_name
 * partner_supporting

Reply:
Ah! I see those on the [SDK Functions](https://developer.parallelmarkets.com/docs/javascript/sdk) page, but not the [SDK Config](https://developer.parallelmarkets.com/docs/javascript/configuration/) page. Should I add them there @bmuller?

Reply:
Actually, I think those are all options that you can send to `login()`, not the SDK config. I don't have any types in here for `login()`, but that'll be quick. I'll tag you on a follow-up, Brian.

Reply:
OK - sorry, think I got the above wrong and will try to clarify here.  I was thinking the ParallelConfig was shared w/ the login options, which I don't think is the case now that I look deeper.

There are a number of params that `login()` can take that will override what was used in the initial `init()` call - but not all and `login` shouldn't accept the same type as `init` - so I think what you had initially is correct.  

The ability for `login()` to take in params like `client_id` that override whatever was initially passed to `init` is an undocumented feature that a few of our partners with unique setups use (I think it should stay undocumented, cause it's only used in very unique / rare cases)

Reply:
Cool. I set the options on `login()` as optional altogether and individually. That should cover all bases. In typical use cases, you'll never pass options to login.